Makefile

```diff
@@ -20,38 +20,42 @@ CUT			=	\033[K
 
 #source files
 SRC_FILES	=	main \
-				Util \
 				CgiHandler \
+				Client \
+				ConfigParser \
+				Connection \
+				ExceptionManager \
+				FileHandler \
 				HttpRequest \
-				HttpRequestParser \
 				HttpRequestHandler \
+				HttpRequestParser \
 				HttpResponse \
-				Socket \
-				Server \
-				ExceptionManager \
 				HttpResponseParser \
-				FileHandler \
-				ConfigParser \
-				Client \
-				Connection \
-				ServerHandler
+				Methods \
+				Server \
+				ServerHandler \
+				Socket \
+				Util
 
-H_FILES =	Exceptions \
-			Util \
+H_FILES =		
 			CgiHandler \
+			Client \
+			ConfigParser \
+			Connection \
+			ExceptionManager \
+			Exceptions \
```

Comment:
Like here, ExceptionManager uses Exceptions so Exceptions should be checked and compiled first and ExceptionManager after that.

Reply:
Alphabetical so I didn't miss anything

Reply:
did it compile?